src/lib/server/eliommod_cookies.ml

```diff
@@ -104,14 +104,14 @@ module Persistent_cookies = struct
     match expiry with
     | Some t ->
         Expiry_dates.add_cookie t cookie >>= fun _ -> Cookies.add cookie content
-    | None -> Lwt.return_unit
+    | None -> Cookies.add cookie content
```

Comment:
I think it would be better if the code was not duplicated: `( match expiry with ... ) >>= fun _ -> Cookies.add ...`.
This will be rewritten to `let _ = match expiry with ... in Cookies.add ...` after the direct-style transition, which looks the most readable.

Reply:
Indeed. Fixed!